src/scrabble.js

```diff
@@ -1,5 +1,67 @@
-function scrabble() {
+// 1.Create a score chart
+// 2.Create a if statment that returns 0 if
+// scrabble('') // should return 0
+// scrabble(" \t\n") // should return 0
+// scrabble(null) // should return 0
+// 3. Somehow loop throught the word and count the letters
+// 4.Count the points
 
+function scrabble(word) {
```

Comment:
Are you going straight for the word? If you're trying to compare each letter to a value I think that 'word' maybe should change to 'letters' instead?